js/stdlib/ts/ALGO.ts

```diff
@@ -1898,32 +1903,47 @@ function ldrop(str: string, char: string) {
 }
 
 /**
- * @description  Format currency by network
- * @param amt  the amount in the {@link atomicUnit} of the network.
+ * @description  Format currency by network or token
+ * @param amt  the amount in the {@link atomicUnit} of the network or token.
  * @param decimals  up to how many decimal places to display in the {@link standardUnit}.
  *   Trailing zeros will be omitted. Excess decimal places will be truncated (not rounded).
  *   This argument defaults to maximum precision.
- * @returns  a string representation of that amount in the {@link standardUnit} for that network.
+ * @returns  a string representation of that amount in the {@link standardUnit} for that network or token.
  * @example  formatCurrency(bigNumberify('100000000')); // => '100'
  * @example  formatCurrency(bigNumberify('9999998799987000')); // => '9999998799.987'
  */
-export function formatCurrency(amt: any, decimals: number = 6): string {
+function handleFormat(amt: any, decimals: number, isNetwork: boolean): string {
```

Comment:
My taste is to take the `splitValue` as an argument explicitly; do you, @keeganthomp , want to write the docs for this too?

Reply:
no problem, I can update this to reflect that.

And for the docs, I sure can. how do I go about doing that?

Reply:
Edit this file around here --- https://github.com/reach-sh/reach-lang/blob/master/docs/src/frontend/index.md?plain=1#L1032-L1052